scripts/bootstrap.py

```diff
@@ -0,0 +1,5 @@
+from aws_data_landing_zone import Scripts
+from data_landing_zone_example_python import config_minimum
```

Comment:
Can you change these imports to have a similar format as in `app.py`

```python
from data_landing_zone_example_python.config_minimum import config
```

and then pass it
```
scripts.boostrap_all(config)
```